INSTALL.md

```diff
@@ -0,0 +1,31 @@
+# TRex install role
+The ansible install-trex role downloads, unarchives, and installs trex on your
+trafficgen host.
+
+## Configfure Ansible inventory file
+Add `trafficgen` host to /etc/ansible/hosts
+```
+sudo echo "trafficgen" >> /etc/ansible/hosts
+```
+
+## Add trafficgen to /etc/hosts
+Replace `127.0.0.1` with the actual trafficgen address
+```
+echo "127.0.0.1 trafficgen" >> /etc/hosts
```

Comment:
What is described here is what an inventory file affords.  Basically, the `ansible-playbook` applies a given playbook to an inventory file.  This let's one write the playbook independent of the inventory against which it applies.  In other words, the `ansible-playbook` provides a "late-binding" of inventory to playbook.